cloudos_cli/__main__.py

```diff
@@ -3606,5 +3610,192 @@ def list_images(ctx,
         click.echo(f"[ERROR] {str(e)}", err=True)
 
 
+@images.command(name="set")
+@click.option('-k',
+              '--apikey',
+              help='Your CloudOS API key.',
+              required=True)
+@click.option('-c',
+              '--cloudos-url',
+              help=(f'The CloudOS url you are trying to access to. Default={CLOUDOS_URL}.'),
+              default=CLOUDOS_URL,
+              required=True)
+@click.option('--procurement-id', help='The specific CloudOS procurement id.', required=True)
+@click.option('--organisation-id', help='The Organisation Id where the change is going to be applied.', required=True)
+@click.option('--image-type', help='The CloudOS resource image type.', required=True)
+@click.option('--provider', help='The cloud provider. Only aws is supported.', required=True)
```

Comment:
why to add this option if only AWS is supported? If there are plans to support other providers soon, I'd add a `type=click.Choice(['aws'])` so the user can only select 'aws' . Also, I'd add a default so the user does not need to add the same 'aws' everytime.